streaming/src/channel/channel.cc

```diff
@@ -292,10 +316,12 @@ StreamingStatus MockProducer::RefreshChannelInfo() {
   MockQueue &mock_queue = MockQueue::GetMockQueue();
   channel_info_.queue_info.consumed_message_id =
       mock_queue.queue_info_map[channel_info_.channel_id].consumed_message_id;
+  channel_info_.queue_info.consumed_bundle_id =
+      mock_queue.queue_info_map[channel_info_.channel_id].consumed_bundle_id;
   return StreamingStatus::OK;
 }
 
-StreamingStatus MockConsumer::ConsumeItemFromChannel(uint8_t *&data, uint32_t &data_size,
+StreamingStatus MockConsumer::ConsumeItemFromChannel(std::shared_ptr<DataBundle> &message,
```

Comment:
According to the code style guide, output parameter `message` should be a pointer instead of a ref.

Reply:
> According to the code style guide, output parameter `message` should be a pointer instead of a ref.

It's unrelated of this function. we shall modify it in next pr.